CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 0.14.0-rc1 / 2015-05-21
```

Comment:
We don't have a 1:1 correlation of master commits to changelog updates though - things are summarized at the end to a more human-readable form, when we actually release. Could be feasible to do a "streaming summary" though - might be annoying though to constantly have to think about it. IMO it's easier to go through everything once before a release, so nothing is forgotten.